bc_obps/reporting/models/base_schema.py

```diff
@@ -0,0 +1,20 @@
+from registration.models import BaseModel
+from django.db import models
+
+
+class BaseSchema(BaseModel):
+    """Static base json schema for emissions reporting forms"""
+
+    # No history needed, these elements are immutable
+    slug = models.CharField(
+        max_length=1000,
+        db_comment="Name of the base schema. Should describe what form it is used to generate and when the base schema took effect. For example: general_stationary_combustion_2024",
+    )
+    schema = models.JSONField(
+        max_length=100000,
```

Comment:
`max_length` is not a valid option for `JSONField` https://docs.djangoproject.com/en/5.0/ref/models/fields/#jsonfield